### DISCUSSION TITLE
Can we constrain it to pack in 2D only ?

### DISCUSSION THREAD

**Discussion by debashis66** [davidmchapman/3DContainerPacking#28] [Q&A]
Can we constrain the packing to happen only in two dimensions ? (LXW, WXH or HXL of container)

**Comment by ghost** [comment]
I think it could work to assign the same value, smaller than any other dimension value, to a selected dimension of both the containers and items? If this was height, then items would only fit "flat" into the container.

**Reply by waynehsmith** [reply]
This is the process that I used (but with a larger dimension (5) in my case. Here is an example:

![image](https://user-images.githubusercontent.com/17620269/150355230-0db817e0-9316-4966-bd29-3d17d4f6678a.png)
